autoware_map_msgs/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+project(autoware_map_msgs)
+
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+set(msg_files
+  "msg/AreaInfo.msg"
+  "msg/PointCloudMapWithID.msg"
+  "srv/LoadPartialPointCloudMap.srv"
+  "srv/LoadDifferentialPointCloudMap.srv")
```

```suggestion
  "srv/GetPartialPointCloudMap.srv"
  "srv/GetDifferentialPointCloudMap.srv")
```
Following up from https://github.com/autowarefoundation/autoware_msgs/pull/39#discussion_r986529866

I think `Get` is easier to understand than `Load`. Because the loading is done in the node but the client is calling this service to `Get` the loaded map.

---

Thanks! Resolved: https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/27a8ea6714aeac6be4f1b394e994aad73fdfa167